src/it/scala/djinni/GeneratorIntegrationTest.scala

```diff
@@ -78,5 +79,33 @@ class GeneratorIntegrationTest extends IntegrationTest with GivenWhenThen {
         assertFileContentEquals(idlFile, OBJCPP, objcppFilenames)
       }
     }
+
+    it("should be able to only generate Java output") {
+      val outputPath = "src/it/resources/result/only_java_out"
+      When("calling the generator with just `--java-out`")
+      val output: String = djinni(s"--idl src/it/resources/all_datatypes.djinni --java-out $outputPath")
```

Comment:
Without the fix, this line throws an exception because the called command returned a non-zero value (1). The tests would fail because of that.